terraform/gcp/variables.tf

```diff
@@ -365,6 +365,12 @@ variable "monitoring_name" {
   default     = "vmmonitoring"
 }
 
+variable "machine_type_monitor_server" {
+  description = "VM size for the monitor server machine"
+  type        = string
+  default     = "custom-1-2048"
```

Comment:
same as it was https://github.com/SUSE/qe-sap-deployment/pull/270/files#diff-3ddf440fbd6f8755a852c2e3bfacc732bd450ffac613b13c0fc12941170fed28L21